cmd/openshift-install/testdata/agent/unconfigured-ignition/configurations/from-ztp-manifests.txt

```diff
@@ -58,14 +58,25 @@ spec:
 
 -- expected/infraenv.yaml --
 metadata:
+  creationTimestamp: null
   name: ostest
   namespace: cluster0
 spec:
   cpuArchitecture: x86_64
+  ipxeScriptType: ""
```

Comment:
The usage of `ipxeScriptType` is unclear to me. The only possible values in assisted are `DiscoveryImageAlways` and `BootOrderControl`. How is the agent installer using these values?

Reply:
I re-checked, and the test changes were triggered by the refactoring/changes in `infraenv.go`